lib/tune.cpp

```diff
@@ -678,6 +680,35 @@ namespace quda
 
   static TimeProfile launchTimer("tuneLaunch");
 
+  using tunepair = std::pair<float, TuneParam>;
```

Comment:
Is this new type needed: can't we use the `time` variable already in the `TuneParam`, and just use a queue of `TuneParam`s?

Reply:
91c29f86284deea69ca9b6c87fdadcb3525a6cb9